src/main/java/vazkii/quark/client/tooltip/EnchantedBookTooltips.java

```diff
@@ -226,9 +226,7 @@ private static void computeTestItems() {
 		testItems = Lists.newArrayList();
 
 		for (String loc : ImprovedTooltipsModule.enchantingStacks) {
-			Item item = ForgeRegistries.ITEMS.getValue(new ResourceLocation(loc));
-			if (item != null)
-				testItems.add(new ItemStack(item));
+			Registry.ITEM.getOptional(new ResourceLocation(loc)).ifPresent(item -> testItems.add(new ItemStack(item)));
```

Comment:
`map(ItemStack::new).ifPresent(testItems::add)`?

Reply:
what

Reply:
cleaner way of doing it